xpsi/utilities/ImportanceSample.py

```diff
@@ -26,31 +26,31 @@
 args = parser.parse_args()
 
 if not args.path_to_mains:
-    args.path_to_mains = input('Specify Absolute or relative path to main files: ')
+    args.path_to_mains = eval(input('Specify Absolute or relative path to main files: '))
```

Comment:
`eval` should not be used for string inputs (paths, filenames, etc.) because it will try to search for a variable with the name of the input string.